File: Java Barcode Scanner ME/app/src/debug/java/de/dimedis/mobileentry/MobileEntryDebugApplication.java
package de.dimedis.mobileentry;

import com.facebook.stetho.Stetho;

public class MobileEntryDebugApplication extends MobileEntryApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        Stetho.initialize(Stetho.newInitializerBuilder(this).enableDumpapp(Stetho.defaultDumperPluginsProvider(this)).enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this)).build());

        DebugUtils.init(this);
        MobileEntryApplication.mDebugUtilsInterface = DebugUtils.getInstance();
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/debug/java/de/dimedis/mobileentry/MobileEntryDebugApplication.kt
package de.dimedis.mobileentry

import com.facebook.stetho.Stetho

abstract class MobileEntryDebugApplication : MobileEntryApplication() {
    override fun onCreate() {
        super.onCreate()
        Stetho.initialize(Stetho.newInitializerBuilder(this)
                .enableDumpapp(Stetho.defaultDumperPluginsProvider(this))
                .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this))
                .build())
        DebugUtils.init(this)
        mDebugUtilsInterface = DebugUtils.instance
    }
}